docs/building-a-form/about-the-json-schema-standard-and-react-jsonschema-form.md

```diff
@@ -0,0 +1,296 @@
+# About the JSON Schema standard and react-jsonschema-form
+
+Learn about the form building code, or *schemaform*, and the library it's built on. For more information about the JSON Schema Standard, see [Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/).
+
+### In this guide
+
+- [Understanding JSON Schema](#understanding-json-schema)
+  - [Describing object fields and arrays](#describing-object-fields-and-arrays)
+- [Understanding react-jsonschema-form (rjsf)](#understanding-react-jsonschema-form-rjsf)
+  - [About the `Form` component](#about-the-form-component)
+  - [Field components](#field-components)
+  - [About the `onChange` field component](#about-the-onchange-field-component)
+  - [About the `uiSchema` field component](#about-the-uischema-field-component)
+- [How schemaform uses rjsf](#how-schemaform-uses-rjsf)
+  - [Customizing fields and widgets from rjsf](customizing-fields-and-widgets-from-rjsf)
+  - [Creating multi-page forms](#creating-multi-page-forms)
+
+### Understanding JSON Schema
+
+The JSON Schema standard describes the allowed shape of JSON objects. You can nest schemas as far down as you'd like. Schemas have a type, which tells you what kind of data is allowed:
+
+```
+{
+  type: 'string'
+}
+```
+
+The JSON schema can also have validation information, such as regexes or length requirements. This example allows any string that's at least two characters and only contains `e` and `f`, meaning `eff` is valid, but `fcc` is not:
+
+```
+{
+  type: 'string',
+  pattern: '^[ef]*$',
+  minLength: 2
+}
+```
+
+You can also specify some built-in `format` values for strings, such as `email`, as a shortcut for including your own patterns.
+
+##### Describing object fields and arrays
+
+This example describes a JSON document that is an object with one property called `myField`, which is a number, meaning `{ myField: 2 }` would be valid:
+
+```
+{
+  type: 'object',
+  properties: {
+    myField: {
+      type: 'number'
+    }
+  }
+}
+```
+
+However, `{}` is also valid. To require a property in an object, use the `required` property. In this example, `required` is on the object that contains the field, not the field itself:
```

Comment:
> In this example

Can `required` be specified on either the object *or* the field?

Reply:
According to the original docs, `required` is always on the object and not the field. So it seems like this should be phrased something like "The `required` field is always on the object..."